.github/actions/stale_issues/action.yml

```
@@ -0,0 +1,49 @@
+name: Filter stale issues
```

It's not filtering a list you give it, but rather finding issues in the repo if I understand correctly?
```suggestion
name: Find stale issues
```